From b3c76fc6b7be5aa2514a268e405e9b6e1c3ebcf0 Mon Sep 17 00:00:00 2001
From: John Crispin <blogic@openwrt.org>
Date: Thu, 30 May 2013 16:16:13 +0200
Subject: [PATCH 128/164] MIPS: ralink: upstream v3.10

patches prior to this were sent upstream and accepted for v3.10

Signed-off-by: John Crispin <blogic@openwrt.org>
---
 dummy |    1 +
 1 file changed, 1 insertion(+)
 create mode 100644 dummy

--- /dev/null
+++ b/dummy
@@ -0,0 +1 @@
+dummy
